docs/tutorials/security/part-3.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Part 3: User trust & management'
+authors: 'Benjamin Fuentes (Marigold)'
+last_update:
+  date: 11 January 2024
+---
+
+> Note : clone this [project](https://github.com/marigold-dev/training-security-3.git) for compiling and testing this tutorial
+
+## Governance :
+
+A decentralized system is not enough to ensure the security and efficiency of a blockchain network. It also requires a robust governance model that can handle conflicts, upgrades, and innovations. Moreover, the distribution of the native token that powers the network should be fair and balanced, avoiding the concentration of power and wealth among a few actors. If these conditions are not met, the decentralized system may suffer from instability, stagnation, or manipulation. Therefore, it is important to design and implement a governance model and a token distribution strategy that align with the goals and values of the network and its users
+
+One of the challenges of designing and deploying a smart contract is to define the roles and permissions of the different parties involved in the execution of the contract. The question is who can do what?
+
+- Who can create, modify, or terminate the contract?
+  - Creation: A smart contract is generally deployed by a DevOps person or a CI pipeline. Apart from knowing the creator's address and the fees he paid, there is no critical event that can appear at this moment. The only hack here would be to impersonate another company's smart contract, as discussed in the introduction of this training
+  - Update: By design, a smart contract code cannot be modified because it is immutable. There is an exception below in the Chapter 2
```

Comment:
```suggestion
  - Update: By design, a smart contract code cannot be modified because it is immutable. However, lambdas are an exception, as described in the next section.
```